apps/showcase/src/app/rules-engine/rules-engine.template.html

```diff
@@ -30,55 +30,54 @@ <h2 id="rules-engine-example">Example</h2>
         Do not hesitate to run the application locally, if not installed yet, follow the <a routerLink="/run-app-locally">instructions</a>.
       </p>
       <a href="https://github.com/AmadeusITGroup/otter/blob/main/apps/showcase/src/components/showcase/rules-engine" target="_blank" rel="noopener">Source code</a>
-      <ul ngbNav class="nav nav-tabs mt-3">
```

Comment:
Why do we delete all this `<ul>`

Reply:
this is already handled by the directive ngbNav using role tablist and role tab